apps/web/src/payload/fields/Inputs/Select/Select.ts

```diff
@@ -0,0 +1,73 @@
+import type { GroupField } from 'payload/types';
+
+function Select({
+  name,
+  interfaceName,
+  fields = []
+}: Partial<GroupField> = {}): GroupField {
+  return {
+    name: name || 'select',
+    type: 'group',
+    interfaceName: interfaceName || 'SelectType',
+    fields: [
+      {
+        label: 'Input Settings',
+        type: 'collapsible',
+        admin: {
+          initCollapsed: true
+        },
+        fields: [
+          {
+            name: 'name',
+            label: 'Name',
+            type: 'text',
+            required: false
+          },
+          {
+            name: 'placeholder',
+            label: 'Placeholder',
+            type: 'text',
+            required: false
+          },
+          {
+            name: 'helpText',
+            label: 'Help Text',
+            type: 'text',
+            required: false
+          },
+          {
+            name: 'label',
+            label: 'Label',
+            type: 'text',
+            required: false
+          },
+          {
+            name: 'required',
+            label: 'Required',
+            type: 'checkbox',
+            defaultValue: false,
+            required: false
+          },
+          {
+            type: 'array',
+            name: 'selectOptions',
+            label: 'Options',
+            required: false,
+            localized: true,
+            fields: [
+              {
```

Comment:
the selectOptions should include a `label` and `value` otherwise the value will default to the label 

```
const options = [
{label: 'name', value: 'customerName'},
{label: 'year', value: 'date'}
]
```